REQUIREMENTS/link-relation-types.md

```diff
@@ -44,13 +44,15 @@ when you think about how you model things in UML: inheritance, implementation, a
 - "Previous": version of the document
```

Comment:
I think this should be lower-case
according to https://www.iana.org/assignments/link-relations/link-relations.xhtml

Reply:
See my other [comment](https://github.com/w3c/wot-architecture/pull/562#discussion_r505625825).